src/Components/ViewComments.jsx

```diff
@@ -1,18 +1,18 @@
-import React from "react";
+// import React from "react";
```

Comment:
How come you've commented everything out here? I say leave it as it was - if it's not being imported anywhere for now that's fine.